binder/views.py

```diff
@@ -1040,6 +1040,31 @@ def _follow_related(self, fieldspec):
 		return (RelatedModel(fieldname, related_model, related_field),) + view._follow_related(fieldspec)
 
 
+	def _pop_limit_and_offset(self, field_where_map) -> (int, int):
```

Comment:
Maybe add a short explanatory comment as to what this function is supposed to do